### DISCUSSION TITLE
How to report these rampant scam hosting their website on github?

### DISCUSSION THREAD

**Discussion by ashishsiwal** [community/community#49217] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi a banking scam is very rampant in India currently where scammers impersonation as bank employees call you and instruct you through their websites and make you install a ".apk" file in your mobile.

As I can find 1 of them and found out that they are using github to host their website. How to report this and get it down?
 Link to scam website : https://instant-e-app-for-campaign.github.io/repo-for-generate/

**Comment by LindseyB** [comment, marked as answer]
Hi, you can contact support https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

**Comment by sanjay09singh** [comment]
Hi, I want to bring attention toward page 
https://snippy.github.io/SipChat

This is used for cyberbullying school kids in Singapore and would request to take down this page ASAP and provide information about the owner of this page with in 7 working days.  

Thanks
